src/libraries/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_11_0/Unicode_11_0_IdnaTest.cs

```diff
@@ -34,52 +34,26 @@ namespace System.Globalization.Tests
     ///
     /// If the value of toUnicode or toAsciiN is the same as source, the column will be blank.
     /// </summary>
-    public class Unicode_11_0_IdnaTest : IConformanceIdnaTest
+    public class Unicode_11_0_IdnaTest : Unicode_IdnaTest
     {
-        public IdnType Type => IdnType.Nontransitional;
-        public string Source { get; set; }
-        public ConformanceIdnaUnicodeTestResult UnicodeResult { get; set; }
-        public ConformanceIdnaTestResult ASCIIResult { get; set; }
-        public int LineNumber { get; set; }
-
         public Unicode_11_0_IdnaTest(string line, int lineNumber)
         {
             var split = line.Split(';');
 
+            // TODO: Fix on NLS, as it doesn't run Non Transitional tests
```

Comment:
>// TODO: Fix on NLS, as it does [](start = 12, length = 31)

could you please track this with issue? or at least we can have issue tracking removing all these tests and writing our own solid tests which not depending on the Unicode data files?

Reply:
Yeah, I'll open an issue for that.